### ISSUE TITLE
[Filter-bar]: [Screen reader should convey ‘Checked’ information for the checked checkbox and ‘Unchecked’ information for the unchecked checkbox both the information]

### ISSUE THREAD

**Issue by sanjayC5373903** [SAP/ui5-webcomponents-react#6120]
### Describe the bug

We are checked with latest version. Screen reader with filter dialog box open. and that dialog box inside for move on next we press Tab Key move arrow Key Up & Down key. but its not working its not moved. and when I am using spacebar for tryed to unchecked checkbox. its not working. It just read text label of checkbox.

Without screen reader with arrow key selection is also working. with screen reader its not working.

We are checked Filterbar on https://sap.github.io/ui5-webcomponents-react
![Screenshot 2024-06-26 151340](https://github.com/user-attachments/assets/fa7318de-1c25-4e36-813c-135ce10df139)
![Screenshot 2024-06-26 152349](https://github.com/user-attachments/assets/d2cc446a-1cfc-4d52-89a5-9b6aef67a0b8)

### Reproduction steps

1. Open Jaws Screen reader.
2. click on filter button. Open filter Dialog box.
3. Press Tab and move on First Row of Checkbox.
4. Then press arrow key up and down. & checked button with spacebar 
5. Then check screen Reader history.

### Expected Behaviour

1. Selection of row should be move up and down with arrow key.
2. Test checkbox should be checked or unchecked with spacebar.
3. Then Screen Reader History should be read specific information of checkbox is checked or unchecked. 

### UI5 Web Components for React Version

~1.29.0

### UI5 Web Components Version

~1.24.0

### Browser

Chrome

### Operating System

Windows11

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by MarcusNotheis** [comment]
Isn't this the exact same issue as in https://github.com/SAP/ui5-webcomponents-react/issues/5979?
Please don't open duplicate issues, this will not speed up the issue resolution.

**Comment by sanjayC5373903** [comment]
Hi @MarcusNotheis , I have opened old issue in last month but i haven't got any response from your teams. because of I thought that raise new issue. Thanks for your response on this issue. please provide necessary help.
